x-pack/plugin/security/cli/build.gradle

```diff
@@ -29,6 +29,11 @@ tasks.named("forbiddenPatterns").configure {
   exclude '**/*.jks'
 }
 
+tasks.named("test").configure {
+  environment 'HOSTNAME', 'dummy.test.hostname' // some tests rely on this being set, which it always is for the code running in main
+  systemProperty 'tests.security.manager', 'false' // the main code under test runs without the SecurityManager
```

Comment:
I understand the point, did the SM cause any issue too ? ( just curious )

Reply:
@jkakavas 
```
Caused by: java.security.AccessControlException: access denied ("java.lang.RuntimePermission" "accessUserInformation")
	at java.base/java.security.AccessControlContext.checkPermission(AccessControlContext.java:485)
	at java.base/java.security.AccessController.checkPermission(AccessController.java:1068)
	at java.base/java.lang.SecurityManager.checkPermission(SecurityManager.java:416)
	at java.base/sun.nio.fs.UnixFileAttributeViews$Posix.checkWriteExtended(UnixFileAttributeViews.java:195)
	at java.base/sun.nio.fs.UnixFileAttributeViews$Posix.setMode(UnixFileAttributeViews.java:264)
	at java.base/sun.nio.fs.UnixFileAttributeViews$Posix.setPermissions(UnixFileAttributeViews.java:299)
	at org.elasticsearch.xpack.security.cli.AutoConfigureNode.execute(AutoConfigureNode.java:219)
```